tests/protocol/mockwiredata.py

```diff
@@ -305,17 +302,18 @@ def mock_http_put(self, url, data, **_):
 
         if url.endswith('/vmAgentLog'):
             self.call_counts['/vmAgentLog'] += 1
-        elif url.endswith('/StatusBlob'):
```

Comment:
"/StatusBlob" is a fake url that was used in the test file that I removed. It's usage was making the put status on the hostgaplugin fail, and forcing the request to be retried as a direct call to storage.

I removed this and added support in the mock protocol for direct and hostgaplugin PUT status requests